### ISSUE TITLE
Bundle error when adding gem

### ISSUE THREAD

**Issue by AssoMania** [refile/refile#551]
Hey there, 
I have this error message on the console when I add the
gem "refile", require: "refile/rails"
gem "refile-mini_magick"
in my gemfile.

I bundle and thats what I get.

Any idea ?
 
```

Bundler could not find compatible versions for gem "rack":
  In Gemfile:
    rails (~> 5.1.4) x86-mingw32 was resolved to 5.1.4, which depends on
      actionpack (= 5.1.4) x86-mingw32 was resolved to 5.1.4, which depends on
        rack (~> 2.0) x86-mingw32

    rails (~> 5.1.4) x86-mingw32 was resolved to 5.1.4, which depends on
      actionpack (= 5.1.4) x86-mingw32 was resolved to 5.1.4, which depends on
        rack (~> 2.0) x86-mingw32

    refile-mini_magick x86-mingw32 was resolved to 0.2.0, which depends on
      refile (~> 0.5) x86-mingw32 was resolved to 0.6.2, which depends on
        sinatra (~> 1.4.5) x86-mingw32 was resolved to 1.4.6, which depends on
          rack (~> 1.4) x86-mingw32

```

I tried to manually add the gem for rack with a version above 2.0. I got the same issue.

Thanks

**Comment by swaps19** [comment]
I got the same issue for `rails 5`.
When I removed `refile-mini_magick` from gemfile, it installed version `0.2.5` for `refile`.
But if I hardcode `refile` to version `0.6.2` which is the current latest, it gives me the compatibility error-

```shell
Bundler could not find compatible versions for gem "sinatra":
  In snapshot (Gemfile.lock):
    sinatra (= 2.0.0)

  In Gemfile:
    sinatra

    refile (~> 0.6.2) was resolved to 0.6.2, which depends on
      sinatra (~> 1.4.5)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

```

I am not getting how can version `0.2.5` be compatible with `sinatra 2.0.0` & version `0.6.2` is not?

**Comment by swaps19** [comment]
Solved the error-
You need to point to the master branch of both gems.
Following worked for me:

```ruby
gem 'refile', require: 'refile/rails', github: 'refile/refile', branch: 'master'
gem 'refile-mini_magick', github: 'refile/refile-mini_magick', branch: 'master'
```

As per many other issues related to the same problem, pointing to the master branch does the job. Hope we'll get a new release soon.
